quickevent/app/quickevent/plugins/Event/src/services/emmaclient.h

```diff
@@ -11,12 +11,13 @@ class EmmaClientSettings : public ServiceSettings
 {
 	using Super = ServiceSettings;
 
-	QF_VARIANTMAP_FIELD2(bool, e, setE, xportTypeXML3, 0)
+	QF_VARIANTMAP_FIELD2(bool, e, setE, xportStartTypeXML3, 0)
```

Comment:
`exportStartTypeIofXml3` or `exportStartListTypeIofXml3` if it is start list

Reply:
exportStartListTypeIofXml3